lib/src/test/java/com/auth0/jwt/PemUtils.java

```diff
@@ -11,10 +12,7 @@
 import java.security.NoSuchAlgorithmException;
 import java.security.PrivateKey;
 import java.security.PublicKey;
-import java.security.spec.EncodedKeySpec;
-import java.security.spec.InvalidKeySpecException;
-import java.security.spec.PKCS8EncodedKeySpec;
-import java.security.spec.X509EncodedKeySpec;
+import java.security.spec.*;
```

Comment:
I prefer the other approach but this is a test so it's fine. However, try running the import organizer shortcut `control+option+O`

Reply:
Fixed.